src/skins/vector/css/matrix-react-sdk/views/dialogs/LocationInputDialog.css

```diff
@@ -0,0 +1,4 @@
+.mx_LocationMap {
```

Comment:
technically this should be two different classes, one for LocationInputDialog and the other for MLocationBody, in order to avoid unnecessarily coupling between the two components and to preserve the naming scheme of the CSS.  I'm happy to turn a blind eye to it here, though, as long as it can be moved into common.css rather than looking like it's specific to LocationInputDialog.css.